c++/src/kj/common.h

```diff
@@ -275,9 +275,8 @@ typedef unsigned char byte;
 #define KJ_UNUSED_MEMBER
 #endif
 
-#if __cplusplus > 201703L || (__clang__  && __clang_major__ >= 9 && __cplusplus >= 201103L)
-// Technically this was only added to C++20 but Clang allows it for >= C++11 and spelunking the
-// attributes manual indicates it first came in with Clang 9.
+#if __cplusplus > 201703L || (__clang__  && __cplusplus >= 201103L)
+// Technically this was only added to C++20 but Clang allows it for >= C++11.
```

Comment:
At this point we require C++20 anyway so we can probably remove this conditional?